src/components/HdTileSelectEditableItem.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <hd-tile-select-item
+    v-if='!isEditing'
+    class="tile-select-editable-item"
+    :class="{
+      'tile-select-editable-item--selected': isSelected,
+    }"
+    v-bind="$attrs"
+    :value="value"
+    @input="enableEditing"
+  />
+  <input
+    v-else
+    class="tile-select-editable-item tile-select-editable-item__input"
```

Comment:
We have a block and an element on the same tag. I think there needs to be made a small adjustment.

Reply:
Nice catch!